packages/core/src/config/load.ts

```diff
@@ -61,6 +50,38 @@ export async function loadConfig(configPath?: string, customExtends?: string[]):
   );
 }
 
+function configTransformer(rawConfig: RawConfig) {
+  if (!rawConfig.apiDefinitions) return rawConfig;
+
+  const config = {} as any;
+  const keysToChange = ['apiDefinitions', 'referenceDocs'];
+
+  for (const [key, value] of Object.entries(rawConfig)) {
+    if (key === 'apiDefinitions') {
+      config.apis = {};
+      for (const [apiName, apiPath] of Object.entries(value)) {
+        config.apis[apiName+'@latest'] = { 'root': apiPath };
+      }
+    }
+    if (key === 'referenceDocs') { config['features.openapi'] = value; }
```

Comment:
Could collide with an existing 'features.openapi' field

Reply:
what is existing 'features.openapi'?

Reply:
I mean, **features.openapi** could be already defined in **redocly.yaml**. And here we are rewriting it with the value of **referenceDocs**.

Reply:
Outdated.